dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -439,10 +453,9 @@ def _get_quantiles(self):
 
         :return: list of quantiles
         """
-        size_bins = 100 / len(self.quantiles)
-        for bin_num in range(len(self.quantiles) - 1):
-            self.quantiles[bin_num] = self._get_percentile(
-                percentile=((bin_num + 1) * size_bins))
+        percentiles = np.linspace(0, 1, len(self.quantiles)+1)[1:]*100
```

Comment:
i'm thinking whether we should put this in the init ?

anyway, we should remove this from the init as it's not used now:
`self.quantiles = { bin_num: None for bin_num in range(1000) }`